lib/shared/config/convert-config.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var path = require('path');
+var logger = require('fancy-log');
+var chalk = require('chalk');
+
+module.exports = {
+
+  description: function(value, configFile) {
```

Comment:
Do we really need the validations on these?  I know it would make it easier to trace but I think it's overkill for a feature this advanced.  Removing this would reduce the code by **a lot** and we could add it back in the future if we find we need it.